src/utils/AppDevUtils.ts

```diff
@@ -61,7 +61,13 @@ const validateURL = (url: string): boolean => {
 
 /* Sort matches by meetingTime in descending order */
 const sortMatchByMeetingTime = (x: Match, y: Match) => {
-  return y.meetingTime.getTime() - x.meetingTime.getTime();
+  const timeX = x.meetingTime;
```

Comment:
what's `timeX` and `timeY`? is it possible to be more descriptive with these namings lool

Reply:
they're just the meetingTime of x and y lol. i guess i can do `meetingTimeX`, but that seems redundant.